### DISCUSSION TITLE
Can I use SeleniumBase built-in methods to add elements directly to the page?

### DISCUSSION THREAD

**Discussion by chenqinggang001** [seleniumbase/SeleniumBase#1429] [Q&A] [answered]
I want to add `<strong>demo</strong>` directly to `<p></p>`

![image](https://user-images.githubusercontent.com/74290987/180607072-765e7958-ebef-4d8a-a1b5-21f08ac08a74.png)

**Comment by mdmintz** [comment, marked as answer]
You'll need to use JavaScript to do that. https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement
With SeleniumBase you can call ``self.execute_script(JS)`` to execute the JavaScript that you need to create elements.

Eg:

```python
from seleniumbase import BaseCase

class AddButtonTest(BaseCase):
    def test_add_button(self):
        script_to_add_button = """function injectButton() {
           var new_button=document.createElement('button');
           document.getElementsByTagName('body')[0].appendChild(new_button);
           new_button.style="width: 200px; height: 120px";
        }
        injectButton();"""
        self.execute_script(script_to_add_button)
        self.sleep(3)
```

**Reply by chenqinggang001** [reply]
Thank you very much😆！
